corgi/tasks/brew.py

```diff
@@ -168,6 +130,70 @@ def slow_fetch_brew_build(
     return build_created or root_created or any_child_created
 
 
+def _get_nested_builds(
+    build_type: str, nested_builds: set[int], force_process: bool, save_product: bool
+) -> None:
+    logger.info("Fetching brew builds for (%s, %s)", nested_builds, build_type)
+    for b_id in nested_builds:
+        logger.info("Requesting fetch of nested build: (%s, %s)", b_id, build_type)
+        slow_fetch_brew_build.delay(
+            b_id, build_type, save_product=save_product, force_process=force_process
+        )
+
+
+def _check_and_load_errata(released_errata_tags, force_process) -> None:
+    """ " for builds with released errata tags set ProductComponentRelation"""
+    for e in released_errata_tags:
+        slow_load_errata.delay(e, force_process=force_process)
```

Comment:
Without the check for missing errata_tags / the separate logger statement, this method is basically a one-liner. It could stay part of the main slow_fetch_brew_build task, unless we expect to reuse it somewhere else.
```suggestion
def _check_and_load_errata(released_errata_tags: list[str], force_process: bool) -> None:
    """For builds with released errata tags, create a ProductComponentRelation for each erratum"""
    for released_erratum in released_errata_tags:
        slow_load_errata.delay(released_erratum, force_process=force_process)
```

Reply:
ok, inlined